README.md

```diff
@@ -64,6 +64,16 @@ You can also supply:
 
 By default this action uses the cf CLI v8 to take advantage of some of the [new features in that version](https://docs.cloudfoundry.org/cf-cli/v8.html#new-workflows). If you need to use v7 of the CLI, you can target the `cli-v7` branch when setting up your workflow, like so: `cloud-gov/cg-cli-tools@cli-v7`
 
+## Protecting secret values in attributes for the deployment
+
+If you have secret values in the attributes (environment variables) of the deployment that should remain secert, it is best to use the `--var` flag with `cf push` in this action so that you can pass a Github secret to the command, which will automatically mask the variable in the deployment logs. Use `cf_command` like so: 
```

Comment:
```suggestion
If you have secret values in the attributes (environment variables) of the deployment that should remain secret, it is best to use the `--var` flag with `cf push` in this action so that you can pass a Github secret to the command, which will automatically mask the variable in the deployment logs. Use `cf_command` like so: 
```